sentry_sdk/tracing.py

```diff
@@ -1290,7 +1292,8 @@ def __init__(
         )  # XXX
         self._active = active
 
-        self._otel_span.set_attribute(SentrySpanAttribute.ORIGIN, origin)
+        if origin:
+            self._otel_span.set_attribute(SentrySpanAttribute.ORIGIN, origin)
```

Comment:
```suggestion
        self.origin = origin
```
Must've missed this in my PRs. There's an `origin` [property](https://github.com/getsentry/sentry-python/pull/3445/files#diff-771cb012ea65fcb826b77e2abb36b504fdf3407fbe8054857b05fe5a41ed5164R1360) now that also checks for `None`

Reply:
yea just pushed that ^^